src/CounterInterceptor/ClientCounter.cs

```diff
@@ -0,0 +1,87 @@
+namespace TemporalioSamples.CounterInterceptor;
+public record ClientCounts
```

Comment:
Should not have a top-level type declared in any non-`Program.cs` file that doesn't match its name. Either make it a nested class or give it its own file.

Reply:
Good catch. Fixed.